modules/quickstart/pages/quick_start.adoc

```diff
@@ -38,118 +36,181 @@ To use Stargate you need:
 // ./mvnw clean package
 // ----
 
-== Getting the Stargate Docker image
+== Pull the Docker image
 
-Pull the Stargate Docker image:
+This image contains the Cassandra Query Language (CQL) and REST APIs with an Apache Cassandra 3.11 backend.
```

Comment:
Doesn't this need the copyright logo after "Apache Cassandra"?

Reply:
Yep, added